individual/tests/graphql_mutation_group_test.py

```diff
@@ -613,11 +624,7 @@ def test_edit_individual_in_group_row_security(self):
             headers={"HTTP_AUTHORIZATION": f"Bearer {self.admin_token}"}
         )
         content = json.loads(response.content)
-        id = content['data']['editIndividualInGroup']['internalId']
-        self.assert_mutation_error(id, 'mutation.individual_group_village_mismatch')
```

Comment:
@delcroip This test case shouldn't have been deleted as it catches the case that an imis admin user can manipulate any individual's group assignment but we should validate to ensure the individual and their assigned group's locations are consistent.